contrib/resources/translations/encode.sh

```diff
@@ -25,41 +31,41 @@ pushd "$trans_dir" > /dev/null || exit
 
 echo "In directory $trans_dir:"
 
-
 # (default)
 # keyb us 437
-# keyb us 858
 tconv_l CP437 "utf-8/en.txt" en
+nconv_t CP437 "utf-8/en.txt" en.utf8
 
 # keyb de
-# keyb de 437
-# keyb de 858
+# keyb de 850
 tconv_l CP850 "utf-8/de.txt" de
+nconv_t UTF-8 "utf-8/de.txt" de.utf8
 
 # keyb es
-# keyb es 437
-# keyb es 858
+# keyb es 850
 tconv_l CP850 "utf-8/es.txt" es
+nconv_t UTF-8 "utf-8/es.txt" es.utf8
 
 # keyb fr
-# keyb fr 437
-# keyb fr 858
+# keyb fr 850
 tconv_l CP850 "utf-8/fr.txt" fr
+nconv_t UTF-8 "utf-8/fr.txt" fr.utf8
 
 # keyb it
-# keyb it 858
+# keyb it 850
 tconv_l CP850 "utf-8/it.txt" it
+nconv_t UTF-8 "utf-8/it.txt" it.utf8
 
 # keyb pl
 # keyb pl 852
-tconv_l CP852 "utf-8/pl.txt" pl
-
 # keyb pl 437
+tconv_l CP852 "utf-8/pl.txt" pl
 tconv_t CP437 "utf-8/pl.txt" pl.cp437
+nconv_t UTF-8 "utf-8/pl.txt" pl.utf8
 
 # keyb ru
 # keyb ru 866
-# keyb ru 808 - CP808 is CP866 with euro sign; iconv does not know 808 (?)
 tconv_l CP866 "utf-8/ru.txt" ru
+nconv_t UTF-8 "utf-8/ru.txt" ru.utf8
```

Comment:
Regarding the non-UTF-8 transliterated files, I'm assuming these will be deprecated.

Is the plan for some stabilization period after which the non-UTF-8 variants would be dropped?

Reply:
I expect they will be dropped. If you wish, I can drop them in this PR, I’ll just make it a separate commit.

Reply:
Dropped the old, non-UTF8 translations.